app/controllers/api/v1/books_controller.rb

```diff
@@ -1,6 +1,9 @@
 module Api
   module V1
     class BooksController < ApplicationController
+
+      before_action :authenticate_user!
```

Comment:
This before action should be in the ApplicationController, all your endpoint (except  the log in one) should be authenticated

Reply:
Done